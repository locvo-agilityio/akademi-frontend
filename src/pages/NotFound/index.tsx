import { Text, VStack } from '@chakra-ui/react';

const NotFound = () => (
  <VStack
    h="full"
    justifyContent="center"
    alignItems="center"
    fontSize="lg"
    fontWeight="bold"
  >
    <Text size="xl">Page Not Found</Text>
  </VStack>
);

export default NotFound;
