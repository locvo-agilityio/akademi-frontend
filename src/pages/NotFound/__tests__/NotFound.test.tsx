import { render } from '@testing-library/react';

import NotFound from '..';

describe('NotFound', () => {
  it('should render successfully', () => {
    const { container } = render(<NotFound />);

    expect(container).toMatchSnapshot();
  });
});
