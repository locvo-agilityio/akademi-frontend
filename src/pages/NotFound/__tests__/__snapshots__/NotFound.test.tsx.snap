// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render successfully 1`] = `
<div>
  <div
    class="chakra-stack css-2ohuy8"
  >
    <p
      class="chakra-text css-0"
    >
      Page Not Found
    </p>
  </div>
</div>
`;
